code/game/objects/items/weapons/shields.dm

```diff
@@ -133,10 +133,16 @@
 	flags_equip_slot = ITEM_SLOT_BACK
 	max_integrity = 300
 	soft_armor = list("melee" = 50, "bullet" = 50, "laser" = 0, "energy" = 100, "bomb" = 30, "bio" = 100, "rad" = 100, "fire" = 0, "acid" = 35)
-	hard_armor = list("melee" = 0, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hard_armor = list("melee" = 5, "bullet" = 5, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
 	force = 20
 	slowdown = 0.2
 
+/obj/item/weapon/shield/riot/marine/equipped(mob/user, slot)
+	. = ..()
+	if(slot == SLOT_BACK && CHECK_BITFIELD(flags_item, NODROP)) //If we're equipping the shield to the back, remove NODROP so it doesn't get stuck there
+		TOGGLE_BITFIELD(flags_item, NODROP)
+		to_chat(user, "<span class='notice'>You loosen the strap of [src] as you place it onto your back.</span>")
+
```

Comment:
This isnt a fix, make it impossible to place nodrop objects in inventory

Reply:
I don't see why it should be prohibited to equip it to the back, and it does fix the fundamental issue with the shield being irremovable, but sure, that's easy enough to do.

Reply:
Done.